src/style.css

```diff
@@ -1,3 +1,95 @@
+/* reset browser default margins */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: var(--main-collor) #000;
+  background-color: #fff;
+}
+/* #region header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo__link {
+  color: var(--main-collor);
+  text-decoration: none;
+  display: block;
+  padding: 10px 0;
+}
+
+.header__logo {
+  width: 40px;
+  margin: auto;
+}
```

Comment:
useless styles

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3483/files#r1253395165
not fixed